vnext/local-cli/runWindows/utils/msbuildtools.js

```diff
@@ -51,18 +51,17 @@ class MSBuildTools {
     newInfo(`Build configuration: ${buildType}`);
     newInfo(`Build platform: ${buildArch}`);
 
-    //const verbosityOption = verbose ? 'normal' : 'quiet';
-    const verbosityOption = 'normal';
+    const verbosityOption = verbose ? 'normal' : 'quiet';
     const args = [
       `/clp:NoSummary;NoItemAndPropertyList;Verbosity=${verbosityOption}`,
       '/nologo',
       `/p:Configuration=${buildType}`,
       `/p:Platform=${buildArch}`,
       '/p:AppxBundle=Never',
+      '/bl',
+      '/flp1:errorsonly;logfile=msbuild.err'
```

Comment:
See the above comment. It feels like we shouldn't need to output a log file. Where would this end up on disk anyway?

Reply:
This ends up in the current working directory.
It is used by runwindows to figure out what msbuild error we hit. Up until now we were saying "build failed with message: undefined". 
This fixes that.

---
In reply to: [405940885](https://github.com/microsoft/react-native-windows/pull/4536#discussion_r405940885) [](ancestors = 405940885)

Reply:
See the comment I have near the top around the logic where we parse this. We should be able to just fix whatever is calling the msbuild process to do the right error reporting.